app/models/location.rb

```diff
@@ -23,6 +23,7 @@ class Location < ApplicationRecord
   scope :search_by, ->(param) { search_by_name(param).or(search_by_postcode(param)) }
   scope :started, -> { where("startdate <= ?", Time.zone.today).or(where(startdate: nil)) }
   scope :active, -> { where(confirmed: true).and(started) }
+  scope :confirmed, -> { where(confirmed: true) }
```

Comment:
I just see them as active record helper methods/filters - totally agree with your second comment, but I put this in as I'll be using it in a couple more places in the [CLDC-2140 PR ](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1455) so that's why I've defined this here to avoid unneccessary diffs